src/pymatgen/io/lammps/inputs.py

```diff
@@ -913,14 +913,18 @@ def write_inputs(self, output_dir: str, **kwargs) -> None:
             output_dir (str): Directory to output the input files.
             **kwargs: kwargs supported by LammpsData.write_file.
         """
-        write_lammps_inputs(
```

Comment:
Need some LAMMPS expert to fix this, this class has only one method (deprecated): https://github.com/materialsproject/pymatgen/blob/0e65d35ae0b1df964639a48067b1a525979eec84/src/pymatgen/io/lammps/inputs.py#L964-L977

And the class itself is not.